src/components/iconbutton/IconButton.module.scss

```
@@ -0,0 +1,25 @@
+@import '../../../styles/common.module.scss';
+@import '../../../styles/colors.module.scss';
```

Please rename the file as `common.scss` and `colors.scss`, colors and common files will not be modules. Also please use the relative path.

```suggestion
@import 'styles/common';
@import 'styles/colors';
```